src/navigation.ts

```diff
@@ -854,7 +855,7 @@ export class Navigation {
    * @returns whether keyboard navigation and editing is currently allowed.
    */
   canCurrentlyEdit(workspace: Blockly.WorkspaceSvg) {
-    return this.canCurrentlyNavigate(workspace) && !workspace.options.readOnly;
+    return this.canCurrentlyNavigate(workspace) && !workspace.isReadOnly();
```

Comment:
Changed for clarity as I had to read the code to learn that `options.readOnly` was mutated when `setIsReadOnly` was called.